test/functorch/test_ops.py

```diff
@@ -1041,6 +1042,10 @@ def fn(inp, *args, **kwargs):
                 xfail(
                     "unbind_copy"
                 ),  # Batching rule not implemented for aten::unbind_copy.int.
+                decorate("linalg.tensorsolve", decorator=skipIfS390X),
+                decorate("nn.functional.max_pool1d", decorator=skipIfS390X),
+                decorate("nn.functional.max_unpool2d", decorator=skipIfS390X),
+                decorate("nn.functional.multilabel_margin_loss", decorator=skipIfS390X),
```

Comment:
This sounds fine to me, but why not submit it first, and then add automation later